pyctcdecode/tests/test_language_model.py

```diff
@@ -68,6 +68,27 @@ def test_trie(self):
         has_token = char_trie.has_node("U.S") > 0
         self.assertTrue(has_token)
 
+    def test_load_from_hub(self):
```

Comment:
This would be a "feature-complete" test. I've uploaded the tiny kenLM model to this model_id on the hub: https://huggingface.co/kensho/testing_dummy_kenlm 

Assuming the tests have internet connection, this would take very little time.

However, if you would prefer to not test from the hub - I can remove this one as well :-)

Reply:
we prefer that tests never call out to the internet. 
 If we really want to be fancy we can write a test and mark it with pytest skip (needs to just be a different method in this test class) but tbh, that might be overkill

Reply:
Sounds good, will remove it!